src/aics-image-viewer/components/App/index.tsx

```diff
@@ -427,30 +429,23 @@ const App: React.FC<AppProps> = (props) => {
 
     imageLoadHandlers.current.forEach((effect) => effect(aimg));
 
-    const initialAlpha = getInitialAlphaLevel();
-    changeViewerSetting("maskAlpha", initialAlpha);
-    view3d.updateMaskAlpha(aimg, alphaSliderToImageValue(initialAlpha));
+    if (doResetMaskAlpha) {
+      const initialAlpha = getInitialAlphaLevel();
+      changeViewerSetting("maskAlpha", initialAlpha);
+      view3d.updateMaskAlpha(aimg, alphaSliderToImageValue(initialAlpha));
+    }
 
     view3d.updateActiveChannels(aimg);
   };
 
-  const onNewVolumeCreated = (aimg: Volume, imageDirectory: string, doResetViewMode: boolean): void => {
-    const channelNames = aimg.imageInfo.channel_names;
-    const newChannelSettings = setChannelStateForNewImage(channelNames);
-
-    setLoadedChannels(new Array(channelNames.length).fill(false));
-    setCurrentlyLoadedImagePath(imageDirectory);
-    changeViewerSetting("viewMode", doResetViewMode ? ViewMode.threeD : viewerSettings.viewMode);
-
-    placeImageInViewer(aimg, newChannelSettings);
-  };
-
   const openImage = async (): Promise<void> => {
     const { fovPath, cellPath, baseUrl } = props;
     const path = viewerSettings.imageType === ImageType.fullField ? fovPath : cellPath;
     const fullUrl = `${baseUrl}${path}`;
 
-    if (path === currentlyLoadedImagePath) {
+    // If this is the same image at a different time, keep luts. If same image at same time, don't bother reloading.
+    const samePath = path === currentImageLoadSpec?.subpath;
+    if (samePath && viewerSettings.time === currentImageLoadSpec?.time) {
```

Comment:
why is there a viewerSettings.time? Should it maybe be viewerSettings.loadSpec.time?  Is the idea of viewerSettings.time to store the value of the time slider?

Reply:
Right, `viewerSettings.time` is the property which would eventually be controlled by the time slider. My thought is that `viewerSettings` is basically the subset of `View3d` and `Volume` properties which are controlled by the UI and so need to be duplicated in React state.

Reply:
ok the only concern here is that we also have a current load spec which also specifies a "current time".

Reply:
I replaced `currentlyLoadedImagePath` state (just a string) with the load spec used to fetch the current image, figuring that was the best way to support more closely examining the specific source of an image and detecting changes other than URL that still require reloading. There may be a better name than `currentImageLoadSpec` that makes this relatively limited purpose clearer?

Reply:
IMO the naming is just fine - the situation with what do we load and from where is just more complicated now. And we are currently set up to only load a single t . The load spec is supposed to encapsulate "everything you need to load the volume data": multiresolution level, time etc.